config/permissions.cfg

```diff
@@ -11,9 +20,9 @@
 #   - create missing user/group if required (default user created: (group: anonymous, password: 12345).
 #   - applicable service, user or group is missing, corresponding permissions are ignored and not updated.
 #   - unknown actions are ignored and corresponding permission are not updated, unspecified action resolves to 'create'.
-#   - already satisfied permission configurations are left as is.
+#   - already satisfied permission configurations are left as is
+#     (ie: existing permission to create or non-existing permission to remove are passed)
```

Comment:
I remember we had a brief discussion about this `permissions.cfg` before.

It can add new perm, update existing perm, but it can not remove existing perm.

Does this line means it can remove existing perm now?

Reply:
It can remove a permission, just need to change to `action: remove`.
It won't remove the service/resource.

Reply:
Cool !